### ISSUE TITLE
Exceptions from `blacksheep/messages.pyx` in version 1.2.19

### ISSUE THREAD

**Issue by waweber** [Neoteroi/BlackSheep#460]
**Describe the bug**

Upgrading to 1.2.19 from 1.2.18 raises several exceptions like:

```
File "blacksheep/messages.pyx", line 302, in blacksheep.messages.Request.host.__get__
    host_header = self.get_first_header(b'host')
  File "blacksheep/messages.pyx", line 46, in blacksheep.messages.Message.get_first_header
    cpdef bytes get_first_header(self, bytes key):
  File "blacksheep/messages.pyx", line 46, in blacksheep.messages.Message.get_first_header
    cpdef bytes get_first_header(self, bytes key):
  File "blacksheep/messages.pyx", line 49, in blacksheep.messages.Message.get_first_header
    for header in self.__headers:

TypeError: 'NoneType' object is not iterable
```

Maybe something went wrong with the Cython compilation?

**Comment by RobertoPrevato** [comment]
Thank You for reporting this issue. I yanked the 1.2.19 release. The problem happened for this reason:

- v1 branch was still compiling and running tests using Cython-0.29.34 (pinned in requirements.txt)
- the release event is handled by the GitHub Workflow, which was modified to build the wheels using cython-3.0.2
- a breaking change in the more recent version of Cython broke the package that way

as a side note, this issue was fixed on v2 branch 4 months ago, at https://github.com/Neoteroi/BlackSheep/commit/d0a9f17f228a6600bcea375d362128214bac1e93

**Comment by RobertoPrevato** [comment]
I published a fix with 1.2.20.
